lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -1073,8 +1079,21 @@ RuleToProcess SessionState::make_rule_to_process(const PolicyRule& rule) {
   RuleToProcess to_process;
   to_process.version = get_current_rule_version(rule.id());
   to_process.rule    = rule;
-  // TODO(@themarwhal): look up teids from PolicyID -> BearerID map
-  to_process.teids = config_.common_context.teids();
+  to_process.teids   = config_.common_context.teids();
```

Comment:
wouldnt it be better to set this as an else in the `if (bearer_exists)`? 

Per what I understand we either use default bearer teid or dedicated bearer teid (?)

Reply:
Yeah maybe I should make it more explicit.